BSP/Simulator/Hardware/PreCharge.py

```diff
@@ -3,10 +3,13 @@
 import os
 import csv
 import fcntl
+import GPIO
  
-file = "BSP/Simulator/Hardware/Data/PreCharge.csv";
-
+file = "BSP/Simulator/Hardware/Data/GPIO.csv";
 
+def write(data, port=2):
+    GPIO_write(data, port)
+    

```

Comment:
There won't be a `GPIO.py`, so this write function should write the data to the correct line of the `GPIO.csv` itself